### PR TITLE
log suggestions

### PR DESCRIPTION
**Pull request by dasdgdafg**
@bearunderscore you will need to update the `spreadsheetId` and also set up google api access via a service account and put the credentials json file as `google api creds.json`

### CODE REVIEW

File: meimei.py [hunk 1]
```diff
@@ -157,5 +176,18 @@ async def getusers(ctx, role: discord.Role, role2: discord.Role):
     print("users in", role2.name, "but not", role.name)
     print(s2.difference(s1))
 
+async def logSuggestion(message):
+    if message.channel.id == 1226401535374655490:
```
Comment (honeycomb547) [inline meimei.py]: maybe put this in a env var, so this can be easily reused

File: meimei.py [hunk 2]
```diff
@@ -8,14 +8,33 @@
...
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+from google.oauth2 import service_account
 
 load_dotenv()
 BOT_TOKEN = os.getenv("BOT_TOKEN")
 bot = commands.Bot(command_prefix="-", intents=discord.Intents.all())
 
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets"]
+credentials = service_account.Credentials.from_service_account_file("google api creds.json", scopes=SCOPES)
```
Comment (honeycomb547) [inline meimei.py:24]: if this bot gets put into a cloud provider we would need to separate the values out into env vars.

**Comment by bearunderscore** [comment]
will push when i have time to set up google api shit

File: meimei.py [hunk 3]
```diff
@@ -138,33 +138,5 @@ async def squish(ctx, image: Optional[Union[discord.PartialEmoji, discord.member
...
-    text = message.content
-    for a in message.attachments:
-        text += a.url
-    result = googleService.spreadsheets().values().append(
-        range=sheet + "!A1", spreadsheetId=SPREADSHEET_ID, valueInputOption="RAW", body={"values":[[text, message.jump_url]]}
-    ).execute()
-    if result['updates']['updatedCells'] > 0:
-        await message.add_reaction("👍")
-        await message.reply("Thank you for the suggestion mister!")
-
```
Comment (dasdgdafg) [inline meimei.py]: These lines should not have been removed when resolving the merge conflicts.

**Comment by bearunderscore** [comment]
THANK YOU KTGD AND AGAIN I"M SORRY IM A FAGGOT WHO CAN"T DO SHIT

**Pull request merged**